packages/website/package.json

```diff
@@ -43,6 +43,10 @@
     "@swc/cli": "0.1.63",
     "@swc/core": "1.3.99"
   },
+  "dependencies": {
```

Comment:
Is this required?

Reply:
We just need to check with Sam if these a actually installed, or are you able to import them because they al already dependencies in the dev server.

Reply:
It seems like they are required as far as I can tell - removing them results in the errors below:

✖ Error importing resolvers/mdResolver.js.
✖ Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'yaml' imported from [...]/lowdefy/packages/website/resolvers/mdResolver.js

Reply:
thanks.